contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer_binary.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+
+from pants.base.build_environment import get_buildroot
+from pants.base.exceptions import TaskError
+from pants.binaries.binary_tool import NativeTool
+from pants.util.process_handler import subprocess
+
+logger = logging.getLogger(__name__)
+
+
+class BuildozerBinary(NativeTool):
+  options_scope = 'buildozer-binary'
+  name = 'buildozer'
+  support_dir = 'scripts/buildozer'
```

Comment:
buildozer is platform-specific, so should really be in bin. bin contains version 0.6.0-1a9c38e0df9397d033a1ca535596de5a7c1cf18f.

Can we bump this to `support_dir = bin/buildozer` and `default_version = '0.6.0-1a9c38e0df9397d033a1ca535596de5a7c1cf18f'`?

(Not really related to your change, but might as well clean up while we're here :))

Reply:
Done.

Reply:
... and reverted. That version causes tests to fail. 

The road to hell is paved with "might as well clean up while we're here"...

Reply:
Ugh, sorry! Will investigate :)